setup.py

```diff
@@ -3,18 +3,23 @@
 #===================================================================================================
 # Imports
 #===================================================================================================
-from distutils.core import setup
+from os import path
+from setuptools import setup
 from vmpooler_client.version import version
 
+def load_readme(readme_filename):
+    return open(path.join(path.dirname(__file__), readme_filename)).read()
+
 #===================================================================================================
 # Main
 #===================================================================================================
 setup(name='vmpooler-client',
-       version=version,
-       description='Manage resources in the vmpooler service from the command-line.',
-       author='Ryan Gard',
-       author_email='ryan.gard@puppetlabs.com',
-       url='https://github.com/puppetlabs/vmpooler-client',
-       packages=['vmpooler_client', 'vmpooler_client.commands'],
-       scripts=['vmpooler_client.py']
+      version=version,
+      description='Manage resources in the vmpooler service from the command-line.',
+      long_description=load_readme("README.rst"),
+      author='Ryan Gard',
+      author_email='ryan.gard@puppetlabs.com',
+      url='https://github.com/puppetlabs/vmpooler-client',
+      packages=['vmpooler_client', 'vmpooler_client.commands'],
+      scripts=['vmpooler_client_app.py']
      )
```

Comment:
Hanging parenthesis are non PEP-8. Yes I know this was my own code so bad on Ryan.